lib/cartopy/crs.py

```diff
@@ -1140,6 +1140,71 @@ def y_limits(self):
         return self._y_limits
 
 
+class LambertEqualArea(Projection):
```

Comment:
This needs to include "Azimuthal" as there is also a [cylindrical alternative](http://www.remotesensing.org/geotiff/proj_list/cylindrical_equal_area.html).

Are you sure this is the equal-area projection you are after?